crates/turborepo-lib/src/lib.rs

```diff
@@ -54,12 +55,18 @@ pub fn get_version() -> &'static str {
 pub fn main() -> Payload {
     match shim::run() {
         Ok(payload) => payload,
+        Err(err @ (Error::MultipleCwd(..) | Error::EmptyCwd { .. })) => {
```

Comment:
Once we delete the Go code, we should route all of them, since it provides a slightly nicer display by default. I'm just doing this to not break all of our integration tests. I'll add a comment noting that